contracts/Oracle.sol

```diff
@@ -10,18 +10,23 @@ contract Oracle is DSMath {
     uint128 constant public turn = 1010000000000000000; // minimum price change 1.01 (1%)
 
     Medianizer med;
-    ERC20   tok;
 
     uint32 public zzz;
     uint32 public lag;
-    address owed;                    // Address owed reward
     uint128 val;                     
-    uint128 lval;                    // Link value
-    uint128 pmt;                     // Payment
-    uint128 dis;
+    uint128 lval;                       // Link value
     uint256 gain;
-    bool posted;                     // Currency price posted
-    bool told;                       // Payment currency price posted
+
+    mapping(bytes32 => Areq) areqs;
+
+    struct Areq {
```

Comment:
Create new Asynchronous Request struct for every call identified by `jobId` ( `bytes32` ).